lib/webclient/features_webclient.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:bytebank/webclient/webclient_config.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:bytebank/models/feature.dart';
+
+final String urlBase = "$urlBaseApi/features";
+
+const _featuresNaoEncontradas = 'Não foram encontradas as features';
+
+//TODO usei o http considerando a sugestão da documentação, tem outra lib recomendada?
+class FeatureWebClient {
+
+  Future<List<Feature>> todas() async {
+    final resposta = await http.get(urlBase);
+    //TODO essa abordagem de verificar o que a resposta indica é normal? Não teria algo como `isSuccessful`?
+    if (resposta.statusCode == 200) {
+      final List jsonDecodificado = json.decode(resposta.body);
+      //TODO vi que tem diversas maneiras de serializar e deserializar json, qual delas recomenda? A que tem o gerador de código?
+      return jsonDecodificado.map((json) => Feature.deJson(json)).toList();
```

Comment:
This is the simplest and built-in way of deserializations.

They all occur on the same (main) thread. If you know it might take a while take and if it is under the scope of this exercise/article a look at `isolates`

Reply:
So I keep this solution and mention the another ways? About the [code generation](https://flutter.dev/docs/development/data-and-backend/json#serializing-json-using-code-generation-libraries), I only link the documentation as a extra exercise?

Reply:
The overhead to add code generators and third-party tools is just that. 
As it is now, I don't see much of a problem, unless this is part of the scope of the project, then it would be fine as an "extra" module.